README.md

```diff
@@ -610,13 +623,20 @@ cabal new-exec summon -- SOME_COMMAND
 cabal new-exec summon-tui -- SOME_COMMAND
 ```
 
-or
+or, if using Stack,
 
 ```shell
 stack exec summon -- SOME_COMMAND
 stack exec summon-tui -- SOME_COMMAND
 ```
 
+of, if using Nix,
+
+```shell
+nix-shell --run 'cabal new-exec summon -- SOME_COMMAND'
+nix-shell --run 'cabal new-exec summon-tui -- SOME_COMMAND'
```

Comment:
Am I right that these commands also will install all required tools for `summoner` (like `git`, `hub` and `curl`) and people can run `summoner` just by this command without needing to install anything except `nix`?

Reply:
That's correct. :-)

---

You'll notice that the last line of `default.nix` is:

```
if pkgs.lib.inNixShell then projectShell else projectDrv 
```

If you run `nix-shell` it will use `projectShell` (thereby all the extra dependencies specified in `buildInputs`); but if you run `nix-build` it will only use `projectDrv` (which is basically haskell packages needed to build the summoner binary).